lib/crawler/keyword/queries/keyword_query.ex

```diff
@@ -0,0 +1,14 @@
+defmodule Crawler.Keyword.Queries.KeywordQuery do
+  import Ecto.Query
+
+  alias Crawler.Keyword.Schemas.Keyword
+
+  def user_keywords_query(user_id),
+    do:
+      Keyword
+      |> where([k], k.user_id == ^user_id)
```

Comment:
can we avoid the `[k]` here

```suggestion
      |> where(user_id == ^user_id)
```

Reply:
@andyduong1920 I'm not sure the syntax here, it doesn't work as I tried. Please correct me!

```elixir
// Compile error
Keyword
      |> where(user_id == ^user_id)
      |> order_by(desc: :inserted_at)

// runtime error
Keyword
      |> where(:user_id == ^user_id)
      |> order_by(desc: :inserted_at)
```

Reply:
oh, I'm not sure too :D